addons/survey/controllers/main.py

```diff
@@ -333,32 +315,27 @@ def survey_reporting(self, survey, token=None, **post):
         #                                         }
         #                      'question': browse record of the question_ids
         #                     }
-        #
         #                     ...
-        #
         #                     ]
         #                 }
-        #
         #             ...
-        #
         #             ]
-        #         }
-        #
-        #     page_range: pager helper function
-        #     current_filters: a list of ids
-        #     filter_display_data: [{'labels': ['a', 'b'], question_text} ...  ]
-        #     filter_finish: boolean => only finished surveys or not
-        #
-
-    def prepare_result_dict(self,survey, current_filters=None):
+        #         },
+        #     'page_range': pager helper function,
+        #     'current_filters': a list of ids,
+        #     'filter_display_data': [{'labels': ['a', 'b'], question_text} ...  ],
+        #     'filter_finish: boolean' => only finished surveys or not,
+        # }
+
```

Comment:
It is required to understand the values sent to template while rendering the survey results.
<b>NO NEED TO REMOVE.</b>